facade/disabledNodeFacade.go

```diff
@@ -31,73 +30,72 @@ var _ = transactionApi.FacadeHandler(&disabledNodeFacade{})
 var _ = validator.FacadeHandler(&disabledNodeFacade{})
 var _ = vmValues.FacadeHandler(&disabledNodeFacade{})
 
-var errNodeStarting = errors.New("Node is starting")
-var emptStr string = ""
+var errNodeStarting = errors.New("node is starting")
+var emptyString string = ""
 
-// nodeFacade represents a facade for grouping the functionality for the node
+// disabledNodeFacade represents a facade with no functionality
 type disabledNodeFacade struct {
 }
 
-// NewDisableNodeFacade creates a new Facade with a NodeWrapper
-func NewDisableNodeFacade() (*disabledNodeFacade, error) {
-	return &disabledNodeFacade{}, nil
+// NewDisabledNodeFacade creates a new Facade with a NodeWrapper
```

Comment:
NewDisabledNodeFacade is the disabled implementation of the facade interface*

Reply:
done